schema.sql

```diff
@@ -30,4 +30,12 @@ CREATE TABLE species (
   ADD COLUMN species_id BIGINT REFERENCES species (id);
   -- Add column owner_id which is a foreign key referencing the owners table
   ALTER TABLE animals
-  ADD COLUMN owner_id BIGINT REFERENCES owners (id);
\ No newline at end of file
+  ADD COLUMN owner_id BIGINT REFERENCES owners (id);
+
+
+-- Create a table named vets
+CREATE TABLE vets(id SERIAL PRIMARY KEY NOT NULL,name VARCHAR(255), age INT,date_of_graduation date);
+-- Create a table named specialization.
+CREATE TABLE specializations(id SERIAL PRIMARY KEY NOT NULL,vets_name VARCHAR(255), species_name VARCHAR(255));
+-- Create a table named visits.
+CREATE TABLE visits(id SERIAL PRIMARY KEY NOT NULL,animals_name VARCHAR(255), vets_name VARCHAR(255),date_of_visit date);
```

Comment:
Since the `visits` table is a join table for `animals` and `vets` it has to have the `Primary key` of both tables. 

Consider doing something like this:

 ```
animal_id int,
vet_id int,
date_of_visit date
```